src/status_im2/contexts/chat/messages/content/text/view.cljs

```diff
@@ -55,11 +56,11 @@
   [blocks {:keys [type ^js literal children]}]
   (case (keyword type)
     :paragraph
-    (conj blocks
-          (reduce
-           render-inline
-           [quo/text]
-           children))
+    (conj (conj blocks
+                (reduce
```

Comment:
This formatting looks odd, maybe just inline them since they're all short names:

```clojure
(conj (conj blocks
            (reduce render-inline [quo/text] children)))
```